app/views/shared/_navbar.html.slim

```diff
@@ -0,0 +1,8 @@
+nav.navbar.navbar-toggleable-md.navbar-inverse.bg-primary
+  .container
+    button.navbar-toggler.navbar-toggler-right[type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation"]
+      span.navbar-toggler-icon
+    a.navbar-brand[href="/"] Personal Finance App
```

Comment:
@sinpai `Personal Finance App` should be moved to a locale. Will be great to do it for all such cases across the app

Reply:
👍